### ISSUE TITLE
strip out "aap-metadata-generator" part from ordinary build process.

### ISSUE THREAD

**Issue by atsushieno** [atsushieno/aap-juce#21]
We run aap-metadata-generator to generate `aap_metadata.xml` from AudioProcessor implementation class. To achieve that, we first build Linux or Darwin version of the plugin, then link it to the extractor console tool.

That takes long time before starting the actual Android project build, while it is only to generate the metadata. And the metadata is actually unchanged unless code changes happen. It is good to run the metadata generator whenever code update occurs, but it actually rarely happens.

Thus, it is probably better to split out metadata generation task from the ordinary builds.

**Comment by atsushieno** [comment]
In aap-lv2 world I actually took that approach for aap-sfizz, and I actually triggered regression for inconsistent updates. So it is desirable to trigger some caution whenever we update the JUCE app submodule if we didn't regenerate metadata.

**Comment by atsushieno** [comment]
It is being worked on as `isolate-metadata-gen` branches in various repos.

**Comment by atsushieno** [comment]
It is merged into main (069844b).
